src/main/java/org/drip/WebUserValidator.java

```diff
@@ -56,6 +56,7 @@ public void validate(Object obj, Errors errors) {
 				errors.rejectValue("email", "email.unique");
 			}
 		}
+		//check if account name is empty
```

Comment:
did this already, lemme check again...maybe am commiting the wrong way...leme check it out